.github/workflows/release.yaml

```diff
@@ -3,7 +3,7 @@ name: release
 on:
   push:
     tags:
-    - v*
+    - "*"
```

Comment:
When I saw the GitHub Actions logs by your description, this part seems not related 👀

Reply:
Oops, I pushed this change by mistake. Thanks for checking.